.github/workflows/linters.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: linters
+
+'on':
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  yamllint:
```

Comment:
Let's not do yaml lint here

Reply:
Should I remove the yaml lint job from here?
yeah, I see linters are failing.

Reply:
Yeah. That's not something we really care about internally

Reply:
Done